CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.1.1 (2021-12-07)
```

Comment:
I can't find the version listed elsewhere within the PR like I usually can.

Reply:
If you're searching within the browser, I think it's lazy loaded due to the high number of changes. It's in [projects/layout/package.json](https://github.com/blackbaud/skyux-layout/blob/9688ef356b5338d2573d5d36ba4aa86dda7ee1ff/projects/layout/package.json).